core/src/main/resources/error/error-classes.json

```diff
@@ -904,6 +904,13 @@
     ],
     "sqlState" : "42000"
   },
+  "NOT_ENOUGH_DATA_COLUMNS" : {
```

Comment:
Is it used somewhere? if not, please, remove it.

Reply:
Done.